src/tests/functional/test_submissions.py

```diff
@@ -67,8 +67,8 @@ def _run_submission_and_add_to_leaderboard(self, competition_zip_path, submissio
         self.find('.item[data-tab="results-tab"]').click()
 
         # The leaderboard table lists our submission
-        prediction_score = Submission.objects.get(pk=submission_id).scores.first().score
-        assert Decimal(self.find('leaderboards table tbody tr:nth-of-type(1) td:nth-of-type(3)').text) == prediction_score
+        prediction_scores = Submission.objects.get(pk=submission_id).scores.first().score
+        assert Decimal(self.find('leaderboards table tbody tr:nth-of-type(1) td:nth-of-type(3)').text) == prediction_scores
```

Comment:
This is the code that was causing the fail selenium tests. I guess I changed it back to what it was originally?
```suggestion
        prediction_score = Submission.objects.get(pk=submission_id).scores.first().score
        assert Decimal(self.find('leaderboards table tbody tr:nth-of-type(1) td:nth-of-type(3)').text) == prediction_score
```